int64 in
---
string message
bool success